android/app/build.gradle

```diff
@@ -220,6 +220,7 @@ dependencies {
     compile "com.google.android.gms:play-services-gcm:8.1.0"
     compile "com.android.support:multidex:1.0.3"
     implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation "me.leolin:ShortcutBadger:1.1.22@aar"
```

Comment:
Are you sure to manually include this dependency?
I guess react-native-notification already does https://github.com/zo0r/react-native-push-notification/blob/master/android/build.gradle
Here where they use the library https://github.com/zo0r/react-native-push-notification/blob/master/android/src/main/java/com/dieam/reactnativepushnotification/helpers/ApplicationBadgeHelper.java